index.html

```diff
@@ -169,9 +265,10 @@ <h1>About me</h1>
 					<p> Hello I’m a software developer! I can help you build a product , feature or website Look through
 						some of my work and experience! If you like what you see and have a project you need coded,
 						don’t hestiate to contact me.</p>
-					<div><button type="button" class="aboutMeButton">Get My Resume</button></div>
+					<div><button type="button" class="aboutMeButton footerTopButton">Get My Resume</button></div>
 				</section>
 			</div>
+			<div class="aboutBackground"></div>
```

Comment:
- I think it will be best if you increase the font size of all the text in your about me container because currently, they look so tiny. 
- The background image looks tiny as well. 🤗 
- You can modify the width of the entire `about container`  a bit, so that all the content are fit closely together like that on Figma.
- Can we kindly fix them?? 🍷 

Here is what I can see, compared to what's on figma 👇🏽 
### Your design
![newhis](https://user-images.githubusercontent.com/99246359/200031707-ed0d12eb-b8b0-4580-b5b5-521fc5bd3fe5.PNG)

### Figma
![abtfig](https://user-images.githubusercontent.com/99246359/200031285-f92b9a96-982b-4c50-a664-8f5eb6f9ef8c.PNG)

Reply:
Thanks for the improvements in my portfolio, sorry I didn't see it it was so bad . I did it like your suggestions.